### DISCUSSION TITLE
class_namespace incorrect?

### DISCUSSION THREAD

**Discussion by imapilot** [livewire/livewire#5946] [7. Bugs (archived)] [answered]
Hello, 

I was getting "Unable to find component:" "ComponentRegistry" error. 

The component would load correctly but when i tried to use a button it threw an error. I did find that the 'class_namespace' => 'App\\Livewire' in the config file livewire.php wasn't as expected. When i updated the config file to 'class_namespace' => 'App\\Http\\Livewire', it started working.

I used the cmd to make:livewire component and it drops it in the App\Http\Livewire folder. 

New laravel 10 project with Jetstream (Livewire) added, then livewire changed to 3.0beta. 

Sorry if its obvious and I'm the issue!

**Comment by joshhanley** [comment, marked as answer]
@imapilot thanks for reporting! In V3, we have moved the default namespace to `App\Livewire`, see here for details https://livewire.laravel.com/docs/upgrading#livewire-app-namespace

As Jetstream hasn't yet been updated for V3, it is still using the old namespace, hence the error.

To solve it, you can either move all the components to the new namespace or publish the config in your app and update `class_namespace` back to the old location.

Hope this helps!

**Reply by imapilot** [reply]
Thankyou!

**Reply by joshhanley** [reply]
@imapilot no worries! 🙂
